.idea/deploymentTargetDropDown.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
shouldn't commit this file, the gradle.xml and the misc.xml files. some of them have values specific to your environment

Reply:
i tried adding it to gitignore but it still pushed it. Can you send me your **gitignore** file

Reply:
my gitignore file would be the same as in the develop branch. i think it might have to do with the fact you already committed the files and it's in the PR. you might have to close this one and open a new PR to develop branch with the new additions in the gitignore file

Reply:
oh son of a gun lol

Reply:
lol sorry